src/widgets-service.js

```diff
@@ -151,7 +158,9 @@ WidgetsService.prototype.createListModel = function (attrs, layer) {
   var widgetModel = new WidgetModel({
     id: attrs.id,
     type: 'list',
-    title: attrs.title
+    title: attrs.title,
+    attrsNames: ['title'],
+    dataviewModelAttrsNames: ['columns', 'columns_title']
```

Comment:
Ok, it is related with list widget. Let's leave it for the moment, but we will need to review it.